src/main/java/reactor/core/publisher/Mono.java

```diff
@@ -2471,12 +2491,12 @@ public T block(Duration timeout) {
 	 * <img class="marble" src="https://raw.githubusercontent.com/reactor/reactor-core/v3.0.6.RELEASE/src/docs/marble/repeatwhen1.png" alt="">
 	 *
 	 * @param maxRepeat the maximum repeat number of time (infinite if {@code Integer.MAX_VALUE})
-	 * @param repeatFactory the
-	 * {@link Function} providing a {@link Flux} signalling the current repeat index from 0 on onComplete and returning a {@link Publisher} companion.
+	 * @param repeatFactory the {@link Function} that returns the associated {@link Publisher}
+	 * companion, given a {@link Flux} that signals each onComplete as a 0-based incrementing {@link Long}.
 	 *
 	 * @return an eventually repeated {@link Mono} on onComplete when the companion {@link Publisher} produces an
 	 * onNext signal
-	 *
+	 * //FIXME
```

Comment:
@smaldini I'm not really sure of the exact behavior of this operator, but the javadoc wording was confusing to me  in any case